Prox/Prox/Models/Place.swift

```diff
@@ -451,6 +451,37 @@ struct OpenHours {
         return nil
     }
 
+    func getEventTimeText(forTime baseTime: Date) -> (String, String)? {
```

Comment:
nit: -> `getEventTimeText(forToday todayDate: Date)`

This method returns "Today" if the given date has an opening date, making the output dependent on the current date - we should make that clear.